pod.go

```diff
@@ -835,6 +814,27 @@ func (p *Pod) start() error {
 		return err
 	}
 
+	l := p.Logger()
+
+	vmStartedCh := make(chan error)
+	const timeout = time.Duration(10) * time.Second
+
+	go p.hypervisor.waitPod(vmStartedCh)
```

Comment:
Can we simply make `waitPod` blocking and pass it a timeout instead?

Reply:
Sure I can do that. The reason I have chosen such an implementation is to avoid the duplication of timeout handling for every hypervisor implementation.